test/test_drakrun.py

```diff
@@ -4,7 +4,14 @@
 @pytest.fixture(scope="session")
 def pytest_installed(drakmon_ssh):
     """ Ensure that pytest is installed """
-    drakmon_ssh.run(". /opt/venvs/drakrun/bin/activate && pip install pytest==6.2.2 pytest-steps==1.7.3")
+    # pip is sometimes broken on virtualenv generated by dh-virtualenv
+    # https://github.com/spotify/dh-virtualenv/issues/338
+    drakmon_ssh.run(". /opt/venvs/drakrun/bin/activate && "
+                    "wget https://bootstrap.pypa.io/get-pip.py &&"
+                    "python3 get-pip.py")
+    # Then we can actually try to install something
+    drakmon_ssh.run(". /opt/venvs/drakrun/bin/activate && "
+                    "pip install pytest==6.2.2 pytest-steps==1.7.3")
```

Comment:
I believe we should include these as part of an install and not do `pip install` in the dh-venv at all

Reply:
I guess pytest and other test dependencies doesn't have to be in the same virtualenv